src/header/header.scss

```diff
@@ -264,5 +292,54 @@
         }
       }
     }
+
+    .iui-split-menu {
+      display: inline-flex;
+      justify-content: center;
+      @include iui-focus;
+
+      svg {
+        width: 16px;
+        @include themed {
+          fill: var(--iui-icons-color-actionable);
+        }
```

Comment:
You can include icons mixin instead.
```suggestion
        @include iui-icons-default;
```

Reply:
`@include iui-icons-default;` uses `@include iui-icons-color();` which is equal to `$iui-icons-color`, not `$iui-icons-color-actionable`.

Reply:
Ah, they have different opacities 😕 

Didn't we remove actionable icons?